draft-ietf-ohai-ohttp.md

```diff
@@ -1253,6 +1323,18 @@ The total number of affected messages affected by server key compromise can be
 limited by regular rotation of server keys.
 
 
+## Client Clock Exposure
+
+Including a `Date` field in requests reveals some information about the client
+clock.  This might be used to fingerprint clients {{UWT}} or to identify clients
+that are vulnerable to attacks that depend on incorrect clocks.
+
+Clients can randomize the value that they provide for `Date` to obscure the true
```

Comment:
Could the Oblivious Gateway Resource also indicate the right time window to the client? This would allow a privacy-sensitive client to randomize the `Date` value correctly to prevent clock fingerprinting, and also reduce chances of rejection.

Reply:
I don't think that that is really necessary.  The idea is to carry the value of `now()`, which the response `Date` field does amply.  Having information about the window (e.g., 10s into the past, 5s into the future) might help if you have very long delays on request/response processing, but I don't see that as being very useful in most cases.  Anything less than 1s will just wash out.